playground/infrastructure/datastore_client.py

```diff
@@ -258,7 +265,7 @@ def _to_snippet_entity(
                 "pipeOpts": self._get_pipeline_options(example),
                 "created": now,
                 "origin": origin,
-                "numberOfFiles": 1,
+                "numberOfFiles": 1 + len(example.tag.files),
```

Comment:
Why are we adding 1? Which file is not represented in `example.tag.files`

Reply:
The meaning of `files` differs in CI/CD and on the backend.
In CI/CD `files` corresponds to a field in `beam-playground` tag, therefore it contains only import files.
The main file is not present here
But on the backend in RunCode request and GetPrecompiledObjectCode response main file code is included in `files` too

Reply:
Thanks for clarifying - could you add a small explanatory comment?

Reply:
```suggestion
                "numberOfFiles": 1 + len(example.tag.files), // add 1 since example.tag only contains import files, not the main file
```

Reply:
Actually, this isn't worth blocking a merge on